proposals/umoci.md

```diff
@@ -0,0 +1,192 @@
+
+   image:
+     config      modifies the image configuration of an OCI image
+     unpack      unpacks a reference into an OCI runtime bundle
+     repack      repacks an OCI runtime bundle into a reference
+     new         creates a blank tagged OCI image
+     tag         creates a new tag in an OCI image
+     remove, rm  removes a tag from an OCI image
+     stat        displays status information of an image manifest
+     insert      insert content into an OCI image
+
+   layout:
+     gc        garbage-collects an OCI image's blobs
+     init      create a new OCI layout
+     list, ls  lists the set of tags in an OCI layout
+
+GLOBAL OPTIONS:
+   --verbose      alias for --log=info
+   --log value    set the log level (debug, info, [warn], error, fatal) (default: "warn")
+   --help, -h     show help
+   --version, -v  print the version
+```
+
+For a more detailed explanation of umoci, see the [project website's guide][umoci-guide].
+
+[umoci]: https://umo.ci/
+[umoci-guide]: https://umo.ci/quick-start/
+
+## Proposal ##
+Change the ownership of the existing umoci project from openSUSE:
+
+  https://github.com/openSUSE/umoci
+
+And move it inside the `opencontainers` organisation:
+
+  https://github.com/opencontainers/umoci
+
+The import paths will correspondingly be "github.com/opencontainers/umoci" (umoci does have some Go API users, but since the project will be renamed -- and GitHub will add a redirect -- there will be no significant downstream impact of the change).
+In the future we may opt to use vanity imports (such as "umo.ci/cmd/umoci").
+
+The project's domain "umo.ci" will also be transferred to the Linux Foundation so that it can be managed by someone other than the maintainers (though maintainers must maintain the necessary administrative access to maintain the website).
+
+### Initial Maintainers ###
+Initial maintainers of the umoci project would be:
+
+* Aleksa Sarai <cyphar@cyphar.com> (@cyphar)
+* Tycho Andersen <tycho@tycho.ws> (@tych0)
+* Vincent Batts <vbatts@hashbangbash.com> (@vbatts)
+
+### Code of Conduct ###
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+[code-of-conduct]: https://github.com/opencontainers/org/blob/master/CODE_OF_CONDUCT.md
+
+### Governance and Releases ###
+This project would initially incorporate the Governance and Release processes from [the OCI project template][oci-template].
+In the future, umoci may choose to modify the Governance and Release processes to better fit the needs of the umoci project.
+
+It should be noted that since umoci is not a specification, it is not bound by the ordinary quorum and voting rules for specification release.
+As such, new versions will be released as regularly as needed without the need for a quorum vote.
+This is to avoid the serious delays in releases encountered within the runc project.
+
+Until there are enough additional maintainers, PRs will be merged with only one maintainer LGTM required.
```

Comment:
Yeah, I wrote this when there was only one maintainer listed and you do have a fair point about potential for abuse. If we allow self-LGTMs then three (or four) relatively active maintainers should be sufficient for a 2 LGTM rule to work. And since the governance processes of a project are largely defined by that project, we can always change it if we feel that it is causing issues.

However one thing that I will change for umoci's governance is the release rule -- I'll update this section to better explain that (the project-template requires three LGTMs for non-spec releases, or a two-thirds vote).

Reply:
Thinking about this some more, I think that the specific concern you had (one maintainer being free to add unlimited maintainers) is not an issue with either version of my proposal -- since adding maintainers has always required a qualified super-majority vote of the maintainers (and that is made explicit in the oci-template governance document). The 2-LGTM rule being relaxed is a separate issue -- and I would argue that technically the TOB shouldn't have an a-priorit opinion on a project's governance rules (though I do understand and appreciate the concern).

So I will update this rule so that there is a mandatory cool-down period (except in the case of security issues) and the rule will have a sunset clause so that it must be explicitly reviewed and approved by a two-thirds majority of maintainers each period of 180 days (or if another maintainer is added before then, immediately).

Reply:
Thanks for updating the proposed governance; IMO it strikes a good balance between the reality of a small maintainer group and baking in some reasonable protections for the future. Thanks.